src/CqlBuilderPanel/ValueSets/Search/Search.tsx

```diff
@@ -160,6 +184,14 @@ export default function Search(props: SearchProps) {
                   prefix="Search"
                   label={SEARCH_MAP[value]}
                   trimField={trimField}
+                  placeHolder={
+                    value == "version" && formik.values.url.length === 0
+                      ? "OID/URL must be entered first"
+                      : ""
+                  }
+                  disabled={
+                    value == "version" ? formik.values.url.length === 0 : false
+                  }
```

Comment:
avoid ==, only ===. 
Same as comment above, you can check to see if a value is truthy like !formik.values.url